solr/core/src/java/org/apache/solr/response/transform/DocTransformers.java

```diff
@@ -49,6 +50,18 @@ public String getName()
     return str.toString();
   }
 
+  @Override
+  public Collection<String> getRawFields() {
+    Collection<String> fields = new ArrayList<>(size());
```

Comment:
this begs to be implemented using Java streams (including flatten).  Just a thought.